docs/pages/access-controls/device-trust/device-management.mdx

```diff
@@ -0,0 +1,172 @@
+enrollment ceremony. To enroll the device, a device enrollment token is
+necessary. The token is created by a device admin and sent to the person
+performing the enrollment off-band (for example, via a corporate chat).
+
+To create an enrollment token run the command below, where `--asset-tag` is
+the serial number of the device we want to enroll:
+
+```code
+$ tctl devices enroll --asset-tag="(=devicetrust.asset_tag=)"
+Run the command below on device "(=devicetrust.asset_tag=)" to enroll it:
+tsh device enroll --token=(=devicetrust.enroll_token=)
+```
+
+## Enroll a trusted device
+
+To perform the enrollment ceremony, using the device specified above, type the
+command printed by `tctl devices enroll`:
+
+```code
+$ tsh device enroll --token=(=devicetrust.enroll_token=)
+Device "(=devicetrust.asset_tag=)"/macOS enrolled
+
+$ tsh logout
+$ tsh login --proxy=(=clusterDefaults.clusterName=) --user=(=clusterDefaults.username=) # fetch new certificates
+Enter password for Teleport user (=clusterDefaults.username=):
+Tap any security key
+Detected security key tap
+> Profile URL:        (=clusterDefaults.clusterName=):443
+  Logged in as:       (=clusterDefaults.username=)
+  Cluster:            (=clusterDefaults.clusterName=)
+  Roles:              access, editor
+  Logins:             (=clusterDefaults.username=)
+  Kubernetes:         enabled
+  Valid until:        2023-06-23 02:47:05 -0300 -03 [valid for 12h0m0s]
+  Extensions:         teleport-device-asset-tag, teleport-device-credential-id, teleport-device-id
+```
+The presence of the `teleport-device-*` extensions shows that the device was
+successfully enrolled and authenticated. The device above is now a trusted device.
+
+## Remove a trusted device
+
+A device that is no longer in use may be removed using `tctl devices rm
+--device-id=<ID>` or `tctl devices rm --asset-tag=<SERIAL>`
+
+First, find a device to delete:
+```code
+$ tctl devices ls
+Asset Tag    OS    Enroll Status Device ID
+------------ ----- ------------- ------------------------------------
+C00AA0AAAA0A macOS enrolled      c9cbb327-68a8-497e-b820-6a4b2bf58269
+```
+
+Now use asset-tag or device id to delete a device:
+```code
+# Delete using asset tag:
+$ tctl devices rm --asset-tag=C00AA0AAAA0A
+Device "C00AA0AAAA0A" removed
+
+# Delete using device id:
+$ tctl devices rm --device-id=c9cbb327-68a8-497e-b820-6a4b2bf58269
+Device "c9cbb327-68a8-497e-b820-6a4b2bf58269" removed
+```
+
+## Dedicated device admin role
+
+We recommend creating a dedicated `device-admin` role for device inventory
+management.
+
+<Admonition type="tip" title="v13.3.6+">
+  Teleport version 13.3.6 and above has the preset `device-admin` role, which
+  is a substitute for the role described below.
+</Admonition>
+
+Following is an example of a role that grants permissions for the `device` resource is necessary to manage
+the inventory. Save the yaml below as `device-admin.yaml` and create it in your
+cluster:
+
+```yaml
+version: v6
```

Comment:
Does role v6 exist in v12 or is that a v13 addition?

Reply:
Hmm, I copied roles from existing v12 doc so I was pretty sure it exists. But your question here and above made me check if we actually backported wrong version to device trust v1 docs - https://goteleport.com/docs/ver/12.x/access-controls/device-trust/guide/#step-12-register-a-trusted-device. 
Another example of `v6` reference in role templates - https://goteleport.com/docs/ver/12.x/access-controls/guides/role-templates/#local-users.

But I found some docs using role `v5` as well. E.g. this role access request doc - https://goteleport.com/docs/ver/12.x/access-controls/access-requests/role-requests/#rbac-setup 

I am confused though 🤔 maybe role `v6` was introduced in v13 and later backported to v12 as well?